docs/resources/aws_iam_users.md

```diff
@@ -48,6 +48,33 @@ The following examples show how to use this InSpec audit resource.
       it { should_not exist }
     end
 
+### Test that all users that have a console password should have used it at-least once
+
+    console_users_with_unused_password = aws_iam_users
```

Comment:
I'd avoid setting a variable if possible - we want to keep inspec examples readable to non-programmers.

```ruby
describe aws_iam_users.where { has_console_password? and password_never_used? } do
  it { should_not exist }
end
```
Might be more readable